lms/views/dashboard/api/grading.py

```diff
@@ -44,43 +42,50 @@ def __init__(self, request) -> None:
         permission=Permissions.GRADE_ASSIGNMENT,
         schema=AutoGradeSyncSchema,
     )
-    def auto_grading_sync(self):
+    def post_auto_grading_sync(self):
```

Comment:
`post_auto_grading_sync` sounds like a method that gets called "after" a grading sync. Something like a hook or event.

Perhaps we should decouple this from the HTTP verb which is supposed to be used when calling it, and instead call it `create_auto_grading_sync` or `schedule_auto_grading_sync`.

Maybe we could even drop the "auto" part from it, as the part that is automatic is contextual. This method only schedules a bunch of grade syncs with the LMS, and could technically be used in other contexts as well.

Reply:
Done :+1: